utils/contributors.js

```diff
@@ -59,6 +59,32 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    {
+        handle_name: 'sachinkg7',
+        name: "Sachin Kumar Gupta",
+        image: "/images/Screenshot 2022-05-30 233414.jpg",
```

Comment:
LGTM but the image is not loading. Try a different image name **without spaces**.